frontend/packages/knative-plugin/src/utils/get-knative-resources.ts

```diff
@@ -56,3 +58,30 @@ export const getKnativeServingServices = (dc: K8sResourceKind, props): KnativeIt
   const ksservices = props && props.ksservices ? getKsResource(dc, props.ksservices) : undefined;
   return ksservices && ksservices.length > 0 ? { ksservices } : undefined;
 };
+
+const getEeventSourceResource = (
+  dc: K8sResourceKind,
+  { eventSourceContainers },
```

Comment:
@invincibleJai What is the difference b/w eventSourceCronJobs and eventSourceContainers here because for both utilities `getEventSourceCronjob` and `getEventSourceContainer` eventSourceContainers are returned?

Reply:
thanks i missed it , will update